packages/application/src/panelhandler.ts

```diff
@@ -90,6 +90,8 @@ export class SidePanelHandler extends PanelHandler {
       this.hide();
     };
     closeButton.className = 'jp-Button jp-SidePanel-collapse';
+    closeButton.title = 'Collapse side panel';
```

Comment:
Should this string be translated?

Reply:
Yes, but I was thinking doing this in a later PR, as it involves more changes.
The `sidepanel`s are created with the `NotebookShell`, so (I guess) before the translator.
Currently the `NotebookShell` does not include a translator, so we should probably add a `translator-extension` that add a translator to the shell, in order to translate the title.
If you think there is a simpler way maybe we can include it in this PR.

Reply:
Ah right. Then we can indeed have a look separately and track this in an issue.

Reply:
Just checked and JupyterLab sets the translator here:

https://github.com/jupyterlab/jupyterlab/blob/e81eb3d5e05e63f162d729066c2f644aa6243a32/packages/translation-extension/src/index.ts#L56-L59

Reply:
Opened https://github.com/jupyter/notebook/issues/6722 to track this.